src/components/email-viewer/email-viewer.tsx

```
@@ -0,0 +1,237 @@
+    }
+
+    private renderAttachments() {
+        if (!this.attachments || this.attachments.length === 0) {
+            return;
+        }
+
+        const label = this.getTranslation('file-viewer.email.attachments');
+
+        return (
+            <div class="attachments">
+                <label>{label}</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)